api_mt/integrations/memoq/memoq.md

```diff
@@ -0,0 +1,213 @@
+## How to set up machine translation plugins in memoQ Translator Pro 9.8™  
```

Comment:
I think we'd be better off with something like this:

```
---
title: How to get machine translation in memoQ
description: Setting up a machine translation plugin in memoQ Translator Pro 9.8™
---

**memoQ** is a translation management system with **machine translation plugins** for many machine translation providers.

A memoQ machine translation plugin requires an API key.
```

And of course similar for the other TMS.

Reply:
Edited as @bittlingmayer suggested for both Trados and memoQ